branchExercise.txt

```diff
@@ -1,3 +1,6 @@
 line 1 bhardwaj first commit
 line 2 bhardwaj first commit
-line 3 bhardwaj first commit
\ No newline at end of file
+line 2.5 neesha second commit
```

Comment:
this should be 2.50...need the zero just 'cause

Reply:
Made the change